### PR TITLE
fix python deploy

### PR DESCRIPTION
**Pull request by yeliang2258**
### PR types

Bug fixes

### PR changes

Others

### Description

1. fix bug in python deploy.

### CODE REVIEW

File: model_zoo/ernie-3.0/deploy/python/ernie_predictor.py [hunk 1]
```diff
@@ -242,9 +242,10 @@ def seq_cls_preprocess(self, input_data: list):
         }
 
     def seq_cls_postprocess(self, infer_data, input_data):
-        infer_data = np.array(infer_data)
-        exp_data = np.exp(infer_data)
-        softmax_data = np.exp(infer_data) / np.sum(exp_data, axis=1)
+        infer_data = np.array(infer_data[0])
+        max_value = np.max(infer_data, axis=1, keepdims=True)
+        exp_data = np.exp(infer_data - max_value)
+        softmax_data = exp_data / np.sum(exp_data, axis=1, keepdims=True)
```
Comment (ZeyuChen) [inline model_zoo/ernie-3.0/deploy/python/ernie_predictor.py]: softmax_data -> probs

File: model_zoo/ernie-3.0/deploy/python/ernie_predictor.py [hunk 2]
```diff
@@ -242,9 +242,10 @@ def seq_cls_preprocess(self, input_data: list):
         }
 
     def seq_cls_postprocess(self, infer_data, input_data):
-        infer_data = np.array(infer_data)
-        exp_data = np.exp(infer_data)
-        softmax_data = np.exp(infer_data) / np.sum(exp_data, axis=1)
+        infer_data = np.array(infer_data[0])
```
Comment (ZeyuChen) [inline model_zoo/ernie-3.0/deploy/python/ernie_predictor.py]: infer_data -> logits

**Review by ZeyuChen — COMMENTED** [review]
更换下变量名

File: model_zoo/ernie-3.0/deploy/python/ernie_predictor.py (hunk 1 above)
Comment (yeliang2258) [reply model_zoo/ernie-3.0/deploy/python/ernie_predictor.py]: Done

File: model_zoo/ernie-3.0/deploy/python/ernie_predictor.py (hunk 2 above)
Comment (yeliang2258) [reply model_zoo/ernie-3.0/deploy/python/ernie_predictor.py]: Done

**Review by ZeyuChen — APPROVED** [review]
